sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -1089,7 +1098,8 @@ <T> Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> queryChangeFeedInter
                 spanName,
                 null,
                 client.getEffectiveDiagnosticsThresholds(
-                    cfOptionsAccessor.getDiagnosticsThresholds(cosmosChangeFeedRequestOptions)));
+                    cfOptionsAccessor.getDiagnosticsThresholds(cosmosChangeFeedRequestOptions)),
+                null);
```

Comment:
why for changeFeed, we do not follow the per trace per page logic?

Reply:
I have eft Changefeed untouched for now - let's discuss in person